stable/prometheus-operator/values.yaml

```diff
@@ -859,6 +865,9 @@ prometheus:
     #   - prometheus.domain.com
     hosts: []
 
+    ## Ingress path, defaults to routePrefix if unset
+    # path: "/"
```

Comment:
The route prefix is also used in several other places. I suggest that you validate the behaviour there as well

Reply:
This doesn't change the behavior of the existing routeprefix and the routeprefix and ingress.path needs to correspond if you choose to override ingress.path. This is to solve the issue with ingress controllers that requires a wild-card (`*`) after the path in order to correctly route traffic to all paths behind the prefix.